airflow/cli/commands/provider_command.py

```diff
@@ -63,4 +69,15 @@ def provider_get(args):
 
 def providers_list(args):
     """Lists all providers at the command line"""
-    print(_tabulate_providers(ProvidersManager().providers.values(), args.output))
+    console = Console()
```

Comment:
The previous version supported different kinds of format output (for example --output grid . We've lost that feature. Can we get it back? Or at the very least we shoud remove the "--output" arg from the command . I am using grid in one of the PRs to check for presence of providers, so it would be nice to keep it.

Reply:
We should be able. I will check.

Reply:
Since one line can contain more than one element, the output is not easily parsed by other scripts.

Reply:
> Since one line can contain more than one element

I agree and that has not changed. The only change here is removing rst syntax from description and adding `|` between columns which helps when using `cut -d "|"`